syntax = "proto3";

package swordsec;

service Swordsec{
 rpc SetUserData(stream UserData) returns (ProcessStatus) {}
}

message UserData {
 int32 id = 1;
 string first_name = 2;
 string last_name = 3;
 string email = 4;
 string gender = 5;
 string ip_address = 6;
 string user_name = 7;
 string agent = 8;
 string country = 9;
}

message ProcessStatus{
 bool status = 1;
}